h2o-r/tests/testdir_jira/runit_pubdev_6262_long_seed_to_string.R

```diff
@@ -0,0 +1,43 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+test.seed.string <- function() {
+    data <- h2o.importFile(path = locate('smalldata/iris/iris.csv'))
+    names(data)[5]<-paste("Species")
+    df <- as.h2o(data)
+
+    # Set parameters
+    params                      <- list()
+    params$y                    <- "Species"
+    params$training_frame       <- df
+    params$ntrees               <- 3
+    params$score_each_iteration <- TRUE
+
+    # Get autogenerated seed as string, it could not be string by default
+    auto_seed <- NULL
+    while(!is.character(auto_seed)){
+        # Train GBM model
+        gbm_model1 <- do.call(h2o.gbm, params)
+        auto_seed <- gbm_model1@parameters$seed
+    }
+    rmse1 <- gbm_model1@model$scoring_history$training_rmse
+    logloss1 <- gbm_model1@model$scoring_history$training_logloss
+
+    # Train model with string seed
+    params$seed <- auto_seed
+    gbm_model2 <- do.call(h2o.gbm, params)
+    rmse2 <- gbm_model2@model$scoring_history$training_rmse
+    logloss2 <- gbm_model2@model$scoring_history$training_logloss
+
+    # Compare results with same string seed setting autogenerated from h2o server
+    expect_equal(auto_seed, gbm_model2@parameters$seed)
```

Comment:
please add a convenience function, eg. h2o.get_seed(model) to retrieve the seed - the function's documentation can mention that the seed can mention the different ways a seed can be represented

Reply:
I added it, however, it does the same functionality as model@parameters$seed right now. Is that what you want?

Reply:
yes, that looks good, please use the new function here in the test